src/components/forms/Select/Select.tsx

```diff
@@ -48,5 +48,3 @@ export const Select = ({
     </select>
   )
 }
-
-export default Select
```

Comment:
I noticed this leading to some new(?) eslint warnings. Since the default export had no usages, and we seem to prefer the named export as a convention, this can be removed 😄